submit.go

```diff
@@ -29,10 +29,10 @@ func (m *CirconusMetrics) submit(output map[string]interface{}, newMetrics map[s
 	numStats, err := m.trapCall(str)
 	if err != nil {
 		m.Log.Printf("[ERROR] %+v\n", err)
+		return
 	}
-	if m.Debug {
-		m.Log.Printf("%d stats sent to %s\n", numStats, m.trapUrl)
-	}
+
+	m.Log.Printf("%d stats sent to %s\n", numStats, m.trapUrl)
```

Comment:
Do you want this unconditionally logged? Seems better to have it in Debug ?  Isn't this noisy?